proto/spacemesh/types.proto

```diff
@@ -128,14 +202,33 @@ message Layer {
     bytes root_state_hash = 5;      // when available - the root state hash of global state in this layer
 }
 
+// current node status
 message NodeStatus {
-    uint64 known_peers = 1;
-    uint64 min_peers = 2;
-    uint64 max_peers = 3;
-    bool   is_synced = 4;
-    uint64 synced_layer = 5;
-    uint64 current_layer = 6;
-    uint64 verified_layer = 7;
+    uint64 connected_peers = 1;       // number of connected neighbors
+    bool   is_synced = 2;             // Is meshed synced
+    uint64 synced_layer = 3;          // the last layer node has synced
+    uint64 top_layer = 4;             // top layer is the tip
+    uint64 verified_layer = 5;        // the last layer node has verified
+}
+
+// Indicates a change in node status
+message NodeStatusEvent {
```

Comment:
You replaced `NodeSyncStatus` with this type, but this type isn't used in any of the services. Also, it's missing

```suggestion
message NodeStatusEvent {
    NodeStatusEvent status = 1;
```